features/authentication-framework/org.wso2.carbon.identity.application.authentication.framework.server.feature/resources/org.wso2.carbon.identity.application.authentication.framework.server.feature.default.json

```diff
@@ -208,5 +208,14 @@
   "authentication.authenticator.iwa_kerberos.parameters.Tags": "Enterprise",
 
   "account_recovery.endpoint.auth.name":"dashboard",
-  "account_recovery.endpoint.auth.hash":"66cd9688a2ae068244ea01e70f0e230f5623b7fa4cdecb65070a09ec06452262"
+  "account_recovery.endpoint.auth.hash":"66cd9688a2ae068244ea01e70f0e230f5623b7fa4cdecb65070a09ec06452262",
+
+  "jit_provisioning.indelible_claims.claim_uris": [
+    "http://wso2.org/claims/userid",
+    "http://wso2.org/claims/username",
```

Comment:
What happens to group claim?

Reply:
Addressed with commit 3498eee331cfdd724a38be7f106ea35897d7ec74